policy/modules/kernel/devices.if

```diff
@@ -2736,103 +2736,95 @@ interface(`dev_dontaudit_getattr_memory_dev',`
 
 ########################################
 ## <summary>
-##	Read raw memory devices (e.g. /dev/mem).
+##	Read raw memory devices (e.g. /dev/mem) if a tunable is set.
+##	This is extremely dangerous as it can bypass the
+##	SELinux protections, and should only be used by trusted
+##	domains.
 ## </summary>
 ## <param name="domain">
 ##	<summary>
 ##	Domain allowed access.
 ##	</summary>
 ## </param>
+## <param name="tunable">
+##	<summary>
+##	Tunable to depend on
+##	</summary>
+## </param>
 #
-interface(`dev_read_raw_memory',`
```

Comment:
All of the original interfaces must remain for backwards compatibility.

Reply:
Ok, reverted